src/cf-layout/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-layout",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
We've been oscillating on the topic but I think we've finally agreed to not update version numbers in PRs. See https://github.com/cfpb/capital-framework/pull/267#discussion_r51621665.

So just leave this at v3.0.3.